napari/layers/labels/labels.py

```diff
@@ -960,7 +954,7 @@ def _update_thumbnail(self):
             # Is there a nicer way to prevent this from getting called?
             return
 
-        image = self._slice.thumbnail.view
+        image = self._slice.thumbnail
```

Comment:
Unsure if I should apply `self._raw_to_displayed` here. Do the input values for `LabelColormap` and/or `DirectColormap` need to be normalized in [0, 1]?

Reply:
They do not! All that happens now is that they get converted to float32 before being sent to a VisPy texture. I think for the thumbnail we don't need to do even that.

Reply:
... But we do need to do the CPU-based colormapping. I am not sure how that happens currently 😅

Reply:
There seems to be some complicated stuff going on with the thumbnail colormapping... Despite the fact that our colormaps have a `map` function that should just handle that. Probbaly could be simplified.

Reply:
> ... But we do need to do the CPU-based colormapping. I am not sure how that happens currently 😅

At with the default colormap, it seems to be working fine. I'm using the `examples/labels3d.py` example. I think that [`Labels._update_thumbnail`](https://github.com/napari/napari/blob/3e3f2ce8c192868bf7db963fed6ce9d7a8a10ee2/napari/layers/labels/labels.py#L935) goes through `LabelsColormap.map` which handles the normalization. From code inspection, it looks like the direct colormap should also be fine, but I haven't tried it yet.

> There seems to be some complicated stuff going on with the thumbnail colormapping... Despite the fact that our colormaps have a `map` function that should just handle that. Probbaly could be simplified.

The extra code is mostly just about using `scipy.ndimage.zoom` to downsample the thumbnail to the desired size.